src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/NullToVisibilityConverter.cs

```diff
@@ -15,7 +15,17 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         {
             if (targetType == typeof(Visibility))
             {
-                return value == null ? Visibility.Collapsed : Visibility.Visible;
+                if (value == null)
+                {
+                    return Visibility.Collapsed;
+                }
+
+                if (CollapseEmptyString && value.GetType() == typeof(string))
+                {
+                    return (string)value == string.Empty ? Visibility.Collapsed : Visibility.Visible;
```

Comment:
```C#
return string.IsNullOrEmpty((string)value) ? Visibility.Collapsed : Visibility.Visible;
```
See [this](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1820?view=vs-2019).

Reply:
I'm adding a change to DetailedPackageMetadata.cs Tags property to be null if null or empty.
Then we can get rid of all these changes to the NullVisibilityConverter.cs.

More elegant way to fix a data layer problem...in the data layer.